src/java/org/apache/cassandra/utils/ByteBufferUtil.java

```diff
@@ -535,6 +535,8 @@ else if (obj instanceof InetAddress)
             return ByteBufferUtil.bytes((InetAddress) obj);
         else if (obj instanceof String)
             return ByteBufferUtil.bytes((String) obj);
+        else if (obj instanceof ByteBuffer)
```

Comment:
spoke about this; its not used in this PR but adds the identity result to this function.  Up to you if you want it or not.

Reply:
leaving this as I was just bitten by it =)